client/src/notebooks/components/StartSessionProgressBar.tsx

```diff
@@ -33,7 +34,7 @@ export interface SessionStatusData {
   details: StatusDetail[];
   message: string;
   readyNumContainers: number;
-  state: string;
+  state: SessionStatusState;
```

Comment:
We could have an enum of string instead, and here the type could be
```suggestion
  state: keyof typeof SessionStatusState;
```